packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/react/whatyouneed.md

```diff
@@ -0,0 +1,2 @@
+* [Download and set up the SDK](/docs/guides/oie-embedded-common-download-setup-app/nodejs/main/)
```

Comment:
Is the node version of this sufficient in the absence of one marked as "for React"?

Reply:
It is not and the work to update the Get Set Up guide is located here: https://oktainc.atlassian.net/browse/OKTA-515730. I'll update this link a part of this Jira Ticket.